src/components/reusable/contactLink/ContactLink.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {FC, JSX} from "react";
+import './contact-link.scss';
+import SvgSprite from "../svgSprite/SvgSprite";
+
+interface TProps {
+    srcType: 'link' | 'phone' | 'email',
+    title: string,
+    src: string,
+    iconId: string,
+    target?: string
+}
+const ContactLink: FC<TProps> = (prop: TProps): JSX.Element  => {
+    const {
+        srcType,
+        title,
+        src,
+        iconId,
+        target = '_blank'
+    } = prop;
+
+    const getSrc = (): string => {
+        let newSrc;
```

Comment:
this variable not needed at all.
![image](https://github.com/Egorkostuck/portfolio/assets/54893185/a5febaed-f649-4a96-844f-7a2eb8161137)